lib/tasks/backend.rake

```diff
@@ -0,0 +1,23 @@
+namespace :backend do
+  desc 'Updates backend_url for a given backend'
+  task modify_url: [:environment] do
+    unless ENV['BACKEND_ID'] && ENV['BACKEND_URL']
```

Comment:
If you use a parameterised rake task, you can call this directly from Jenkins using the Run rake task job:
https://www.viget.com/articles/protip-passing-parameters-to-your-rake-tasks
https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/